test-cypress/integration/repository/repositories.spec.js

```diff
@@ -503,6 +505,44 @@ describe('Repositories', () => {
         ModalDialogSteps.verifyDialogBody('Changing the repository ID is a dangerous operation since it renames the repository folder and enforces repository shutdown.');
     });
 
+    it('should not allow restart of local repository, if node is in cluster', () => {
+        // Given I create a repository
+        cy.createRepository({id: repositoryId});
+        // When I set the node in a cluster
+        GlobalOperationsStatusesStub.stubGlobalOperationsStatusesResponse(repositoryId);
+        //ClusterStubs.stubClusterNodeStatus();
+        // Then go to the edit repository page
+        RepositorySteps.visitEditPage(repositoryId);
+        // I expect the repository restart button to be disabled
+        RepositorySteps.getEditViewRestartButton().should('be.visible');
+        RepositorySteps.getEditViewRestartButton().should('be.disabled');
+    });
+
+    it('should not allow restart of local repositories, if node is in cluster', () => {
+        // Given I create a repository
+        cy.createRepository({id: repositoryId});
+        // When I set the node in a cluster
+        GlobalOperationsStatusesStub.stubGlobalOperationsStatusesResponse(repositoryId);
+        ClusterStubs.stubClusterNodeStatus();
+        // Then go to the repositories page
+        RepositorySteps.visit();
+        // I expect the repository restart button to be disabled
+        RepositorySteps.getRepositoryRestartButton(repositoryId).should('be.visible');
+        RepositorySteps.getRepositoryRestartButton(repositoryId).should('be.disabled');
+    });
+
+    it('should allow restart of remote repositories, even if node is in cluster', () => {
```

Comment:
Describe this test better, please, because for a person who is unfamiliar with this behavior it won't be clear why this test verifies the permission to restart in cluster and the previous two tests are doing the oppisite.